cdk/lib/admin-console.ts

```diff
@@ -253,10 +253,13 @@ export class AdminConsole extends GuStack {
       archivedBannerDesignsDynamoTable,
     );
 
-    const userData = `#!/bin/bash -ev
-    aws --region ${this.region} s3 cp s3://membership-dist/${this.stack}/${this.stage}/${app}/support-admin-console_1.0-SNAPSHOT_all.deb /tmp
-    dpkg -i /tmp/support-admin-console_1.0-SNAPSHOT_all.deb
-    /opt/cloudwatch-logs/configure-logs application ${this.stack} ${this.stage} ${app} /var/log/support-admin-console/application.log`;
+    const userData = UserData.forLinux();
```

Comment:
The way we define the userData has changed, see https://github.com/guardian/cdk/pull/2378